code/modules/mob/living/carbon/carbon_defense.dm

```diff
@@ -479,6 +479,9 @@
 		//SKYRAT EDIT ADDITION
 		if(HAS_TRAIT(src, TRAIT_OVERSIZED) && !HAS_TRAIT(M, TRAIT_OVERSIZED))
 			visible_message(span_warning("[M] tries to pat [src] on the head, but can't reach!"))
+		else if(HAS_TRAIT(src, TRAIT_UNDERSIZED) && !HAS_TRAIT(M, TRAIT_UNDERSIZED))
+			visible_message(span_warning("[M] tries to pat [src] on the head, a loud cracking noise coming from their back as they bend down!"))
+			M.apply_damage(1, BRUTE, BODY_ZONE_CHEST)
```

Comment:
This is meme-y and shouldn't exist. You shouldn't get the damage from iron ass either but that's just a case of no one caring enough to remove it. There's no reason to punish people for... trying to pat a small person.
You can pick up a screwdriver off the ground, but your back creaks like the Titanic's hull because you tried to pat something the size of a cat, seriously?

Reply:
I did it because I had done some changes to the one above it first, and was bored after having gotten no feedback on this ages. But sure, if you don't think that 1 damage as a joke is funny, then it can be removed.